cumulus/client/service/Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "cumulus-client-service"
 version = "0.1.0"
 authors.workspace = true
 edition.workspace = true
+license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
```

Comment:
Only the crates in cumulus/client have the correct license. All the other licenses need to be fixed.

Reply:
Can you list the correct ones so I can get them fixed?

Reply:
All crates in `client/cumulus` (the path) are correct and the other ones should be changed to APACHE 2.0